samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -60,6 +60,11 @@ public class ChatSkill
     /// </summary>
     private readonly PlannerOptions _plannerOptions;
 
+    /// <summary>
```

Comment:
@TaoChenOSU this is my work around since the context variables aren't being populated as expected

Reply:
Can you let me know more about the issues you are having, especially if you think they are an issue with `Plan`? Thanks!

Reply:
The issue we were seeing is that the context variables aren't being propagated down the prompt chain as expected. 

It's not specifically a planner issue, but when I tried setting a new context variable 'proposedPlan' in AcquireExternalInformation method, it's not returned in the chatContext response in the ChatAsync method. 

Another observation to back this is the token limits aren't being passed down, i.e.,
tokenLimit is set in ExtractUserMemoriesAsync, so it should be lower or equal to that value in following methods, but it's not. 

One run yielded:
- tokenLimit = 2829 at the end of ExtractUserMemoriesAsync
- then tokenLimit = 2846 in AcquireExternalInformationAsync (which is called after)